MekHQ/src/mekhq/gui/baseComponents/AbstractMHQNagDialog.java

```diff
@@ -109,7 +109,7 @@ public AbstractMHQNagDialog(final Campaign campaign, final String nagKey) {
         leftBox.setAlignmentX(Component.LEFT_ALIGNMENT);
 
         // Get speaker details
-        Person speaker = campaign.getSeniorAdminCommandPerson();
+        Person speaker = campaign.getSeniorAdminPerson(0);
```

Comment:
Along with promoting the above Admin Type constants, this should become:

`Person speaker = campaign.getSeniorAdminPerson(Campaign.ADMIN_TYPE_COMMAND);`

to avoid introducing magic numbers.